code/__HELPERS/logging.dm

```diff
@@ -36,6 +36,22 @@
 	GLOB.STUI.admin.Add("\[[time_stamp()]]ADMIN: [text]")
 	GLOB.STUI.processing |= STUI_LOG_ADMIN
 
+/proc/check_slur(var/text, var/mob/source, var/method = "Speech")
+	if(!CONFIG_GET(flag/log_slurs))
```

Comment:
this is called log_slurs but it prevents checking slurs altogether. either rename it or remove it

Reply:
will probably rename it to block_slurs when I make next changes